src/App.tsx

```diff
@@ -3,29 +3,66 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
+
+import { Todo } from './types/Todo';
 
 interface State {
   selectedUserId: number;
+  todos: Todo[] | [],
```

Comment:
Illia said there is no need for empty arrays)
```suggestion
  todos: Todo[],
```

Reply:
Yeah, empty array will still work with `Todo[]` type